TalentBoard/talentBoard.js

```diff
@@ -29,7 +29,14 @@ async function addProfile(event) {
   if (validationResult.length) return badRequestResponse(validationResult);
 
   // Calculating expireAfter field (expireAfter field will keep track as to when the profile should expire)
-  const days = event.visibilityDuration;
+  let days = 0;
+
+  if (event.visibilityDuration === 1) days = 15;
```

Comment:
instead, create a constants variable (`visibilityDurationMapping`) with the key being the `visibilityDuration` and value being the corresponding values and then, you can have this:

```JS
const days = visibilityDurationMapping[event.visibilityDuraion];
```

By this, you can eliminate the entire `if-else` ladder here

Reply:
done